src/crt_launch/crt_launch.c

```diff
@@ -85,6 +85,7 @@ struct host {
 };
 
 static int	my_rank;
+volatile static int	myflag = 0;
```

Comment:
(style)  do not initialise statics to 0
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt
(style)  storage class should be at the beginning of the declaration